data/Store.ts

```diff
@@ -692,6 +693,8 @@ export class Store extends HoistBase {
      * for backwards compat with app code predating support for multiple {@link summaryRecords}.
      */
     get summaryRecord(): StoreRecord {
+        if (isNull(this.summaryRecords)) return null;
```

Comment:
Nit - would be fine to just check if `summaryRecords` is truthy if you wanted to just write `if (!this.summaryRecords)`, but I believe what you have is correct since `summaryRecords` cannot be `undefined`.

Reply:
I'm also a frequent user of lodash `isEmpty` - would be usable here, as we want to return `null` if `summaryRecords` is nullish or empty.